api/api.yaml

```diff
@@ -1,61 +1,90 @@
-swagger: "2.0"
+openapi: 3.0.0
```

Comment:
![Screen Shot 2019-03-27 at 1 35 44 PM](https://user-images.githubusercontent.com/1866852/55080019-49cc8480-5095-11e9-9cd2-ea22642c3c13.png)
There is an error

Reply:
yes i was just about to tell you that error is persisiting no matter what i change looked line by line even had kevin look at it for me

Reply:
?

Reply:
i don't understand

Reply:
you dont understand what ?

i am just asking if that has been fixed

Reply:
?